### PR TITLE
POS Bookings: Avatar placeholder for team members

### PR DESCRIPTION
**Pull request by staskus**
Closes: WOOMOB-2260

## Description

Add avatar support for team members in booking rows:
- Show team member photo with a border when an image URL is available  (using KFImage caching)
- Show initials placeholder when a team member has no image
- Show nothing when no team member is assigned

## Screenshots

| Light mode | Dark mode |
|---|---|
|  |  |

## Testing instructions

1. Set up team members in the Bookings admin (some with profile images, some without)
2. Assign team members to bookings
3. Open POS and navigate to Bookings
4. Confirm:
   - Team members with images show a circular photo with a border
   - Team members without images show an initials placeholder
   - Bookings without a team member show no avatar
   - Verify in both light and dark mode

### CODE REVIEW

File: Modules/Sources/PointOfSale/Presentation/Bookings/POSBookingAvatarView.swift [hunk 1]
```diff
@@ -1,31 +1,71 @@
...
+        }
+    }
+
+    static func initials(from name: String?) -> String? {
+        guard let name, !name.isEmpty else { return nil }
+        let components = name.split(separator: " ")
+        if components.count >= 2 {
+            return String(components[0].prefix(1) + components[1].prefix(1)).uppercased()
         }
+        return String(components[0].prefix(1)).uppercased()
```
Comment (toupper) [inline Modules/Sources/PointOfSale/Presentation/Bookings/POSBookingAvatarView.swift]: I am wondering here, what if the name contains only whitespaces? eg. "     ". Would it crash if `split(separator: " ") `returns an empty array? Could we guard on `components.first` before indexing to be safe here?

File: Modules/Sources/PointOfSale/Presentation/Bookings/POSBookingAvatarView.swift [hunk 2]
```diff
@@ -1,31 +1,71 @@
...
+                .frame(width: Constants.avatarSize, height: Constants.avatarSize)
+                .overlay(
+                    Text(initials)
+                        .font(.system(size: Constants.initialsSize, weight: .medium))
+                        .foregroundStyle(textColor)
+                )
+        }
+    }
+
+    static func initials(from name: String?) -> String? {
```
Comment (toupper) [inline Modules/Sources/PointOfSale/Presentation/Bookings/POSBookingAvatarView.swift]: I’m wondering if this helper should live outside the view and get unit tests, since it’s pure formatting logic and now has a few edge cases to protect (whitespace-only names, multi-word names, etc.). Keeping it inside the view makes it harder to test and easier to regress.

**Review by toupper — COMMENTED** [review]
Great addition @staskus! It looks good to me, but before approving it would be awesome if you could take a look to ensure that we are crash safe.

File: Modules/Sources/PointOfSale/Presentation/Bookings/POSBookingAvatarView.swift (hunk 1 above)
Comment (staskus) [reply Modules/Sources/PointOfSale/Presentation/Bookings/POSBookingAvatarView.swift]: Great catch! And writing unit tests for it would probably have caught such an issue. Always need to be careful with []... Thanks again for a careful look.

File: Modules/Sources/PointOfSale/Presentation/Bookings/POSBookingAvatarView.swift (hunk 2 above)
Comment (staskus) [reply Modules/Sources/PointOfSale/Presentation/Bookings/POSBookingAvatarView.swift]: Done! Extracted into `POSAvatarInitialsFormatter` with unit tests covering all the edge cases (nil, empty, whitespace-only, single word, multi-word, leading/trailing spaces, etc.).

**Review by toupper — APPROVED** [review]
Awesome, thanks a lot for the changes! LGTM 🚢 :shipit:
